python-manual/modules/ROOT/pages/query-simple.adoc

```diff
@@ -151,7 +151,7 @@ driver.execute_query(
 )
 ----
 
-**None of your keyword query parameters may end with a single underscore.** This is to avoid collisions with the keyword configuration parameters. If you need to use such parameter names, pass them in the `parameters_` dictionary.
```

Comment:
Did you revert this line on purpose_

Reply:
No, just a sloppy conflict resolution :hankey: 

PS. `execute_query` kwarg style all the way ________ :rofl:

Reply:
You know, important things just need to be underscored sometimes :rofl: